xtask/src/build_docker_image.rs

```diff
@@ -289,7 +289,7 @@ pub fn determine_image_name(
             }
             tags.push(target.image_name(repository, version));
             // Check for unstable releases, tag stable releases as `latest`
-            if version.contains('-') {
+            if !version.contains('-') {
                 // TODO: Don't tag if version is older than currently released version.
                 tags.push(target.image_name(repository, "latest"))
             }
```

Comment:
don't know how to best solve the issue of this taging backports as latest.

Should we support backports? Do we need to?

Reply:
We might need to if we need to say, fix the `0.2` branch and we have released `0.3`, I believe.

Reply:
this is implemented now